cluster/seq-cluster-docker-compose/compose.yaml

```diff
@@ -53,9 +96,10 @@ services:
     networks:
       - seqclusterdemo
     ports:
-      - 80:80
+      - "5777:80"
```

Comment:
80 is a pain to bind in some configurations. Fails for me with Podman on macOS running unprivileged.